.travis.yml

```diff
@@ -14,8 +14,14 @@ addons:
       - debian-sid
     packages:
       - shellcheck
-install: tests/install.sh
-script: tests/${TEST}
+install:
+  - tests/install.sh
+  - pip install pytest-cov
+script:
+  - tests/${TEST}
+  - py.test --cov=cinch/bin --cov-config .coveragerc --cov-report term --cov-report xml --cov-report html tests
```

Comment:
This will run the coverage test at least 4 times for each commit. Instead, you should add this line to a new shell script under the tests/ directory and then add that shell script to the env matrix.